content/features/featuresDeepDive/materials/node_material/nodeMaterial.md

```diff
@@ -426,6 +427,22 @@ If `ALPHATEST = 0`, the computed value is `-1 + alphaCutOff`. As `alphaCutOff` i
 
 You could also have used `Lerp(0, alphaCutOff, ALPHATEST)` as the input for `Discard.cutoff`, but it's likely that the addition + subtraction used above is faster than a `Lerp` on GPUs (would need some benchmarking to be sure), even if it's by a small (negligible) margin.
 
+#### Using loops
+
+Loops are always a complicated discussion within a shader as they can easily kill the performance. With Babylon v8.0, we are introducing a new node named `Loop`. The goal of that node is to allow the user to aggregate a value through iterations (Think for instance about a texture blur).
+
+To do so you will need to drop a `Loop" block and connect an input value. That value will be the initial value of the aggregated variable. Once computed that value will be available on the output port of the block.
+
+To aggregate the value you will need to wire a `StorageRead` and `StorageWrite` block to the loopID port of the `Loop` block. These 2 blocks will give you access to the running value of the aggregated variable.
+
+Here is an example where the loop is simply adding 2 to a float 10 times:
+![Emissive](/img/playgroundsAndNMEs/simple-loop.png)

```

Comment:
[Emissive] should be [Simple loop]. This is likely an overlooked copy/paste as the referenced image is incrementing alpha. For the image linked here, the float being added in the loop does not display 2.0, it displays 0.0 so it does not match with the text. It also seems strange to pump this number into alpha as the user won't see the change. Maybe use the initial value of 0.0 and an iterated value of 0.05. This will result in a visible outcome the user can understand.

Maybe this simple example should also be a PG just so the user can test a very simple loop.